charts/timber-fluentd/values.yaml

```diff
@@ -0,0 +1,124 @@
+global:
+  # -- Extra pod labels in a map[string]string format, most likely to be used for the costing labels.
+  extraPodLabels: {}
+
+nameOverride: ""
+
+image:
+  # -- Docker registry for fluentd image
+  registry: ghcr.io
+  # -- Docker image repository for fluentd
+  repository: caraml-dev/timber/fluentd
+  # -- Docker image tag for fluentd
+  tag: v0.0.0-build.16-01ac82e
+  # -- Docker image pull policy
+  pullPolicy: IfNotPresent
+
+replicaCount: 1
+
+# -- Resources requests and limits for fluentd StatefulSet. This should be set
+# according to your cluster capacity and service level objectives.
+# Reference: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+resources: {}
+
+# GCP configurations for flushing Observation logs to BQ
+gcpServiceAccount:
```

Comment:
sensitive data should be provided in a secret (that is in place already) and that secret can be provided from the outside. 
for example:
1. no secret provided
```
gcpServiceAccount:
  credentialsData: ...
```
2. secret provided:
```
gcpServiceAccount:
  credentials:
    name: secret-name
    key: sa.json
```
3. workload identity:
```
gcpServiceAccount: nil
```
and this should be a default option for deployment. to do so, we also need to have a kubernetes service account that can be linked to GSA

Reply:
for the 3rd point (presumedly default) i added a `kubernatesServiceAccount` which will be appended to the pod 
```
spec:
  serviceAccountName: {{ .Values.kubernatesServiceAccount }}
```

Please review 🙏